### PR TITLE
Russian translation update

### PR DESCRIPTION
**Pull request by nexfwall**
It looks like you used Google Translate or something similar to translate that file to Russian.

So now translations looks more correct, from the point of view of some Russian guy from the internet.

### CODE REVIEW

**Comment by Adaptivity** [comment]
What the hell? I say "no" to this PR. *from the point of view of some Russian guy from the internet.* exactly, from the random guy. You can use this as resource pack or just replace .lang file in the .jar or whatever you want but *such translation* will not be pulled.

**Comment by nexfwall** [comment]
@Adaptivity
First off I think my PR will be ignored too, like yours, which you made a month ago.

And if you can only tell me how are you frustrated, without pointing on mistakes which I missed or telling me about what's wrong, I have nothing to say you more.

**Comment by Adaptivity** [comment]
My PR is not ignored. It will be merged when TE will be updated. But yours... hell no. You are fully broke .lang file. [The only change I'm ok with](https://github.com/nexfwall/ThermalExpansion-Localization/commit/c9874195ee69e68e582e60e69407fd7337305abc). I just probably forgot to change this during update to 1.7.10.

**Comment by nexfwall** [comment]
I had only one aim: to translate the phrases as I see fit. What would provide some sense in this. Because this translation is made by machine, i see it.

Тебе ведь знаком термин "Надмозг"? Вот только так я могу объяснить этот перевод.

**Comment by nexfwall** [comment]
Okay. Do what you need.

**Comment by nexfwall** [comment]
@Adaptivity
And also

[This is a spelling mistake, isn't it?](https://github.com/CoFH/ThermalExpansion-Localization/pull/27/files#diff-f70b5daf3346760858bf7522565b3cf1R31)

**Comment by Adaptivity** [comment]
You made this change... it's your mistake.

File: ru_RU.lang [hunk 1]
```diff
@@ -1,63 +1,63 @@
...
-achievement.thermalexpansion.transposer=(В основном) Бесплатное заполнение
-achievement.thermalexpansion.transposer.desc=Заполнять вёдра руками - тяжёлая работа.
-achievement.thermalexpansion.precipitator=Снежки!
+achievement.thermalexpansion.pulverizer.desc=Дроблёная руда.
+achievement.thermalexpansion.sawmill=Ненавижу деревья, но люблю древесину
+achievement.thermalexpansion.sawmill.desc=Прям как рубка деревьев, но лучше.
+achievement.thermalexpansion.smelter=Два слитка входит, один выходит
+achievement.thermalexpansion.smelter.desc=Точно не съедобно.
+achievement.thermalexpansion.crucible=Расплавляйте ВСЁ
+achievement.thermalexpansion.crucible.desc=Да, она может делать лаву. Вы монстр.
```
Comment (Adaptivity) [inline ru_RU.lang]: Магмовый тигель -> тигель мужского рода.

File: ru_RU.lang [hunk 2]
```diff
@@ -1,63 +1,63 @@
...
+achievement.thermalexpansion.transposer=(Почти) Бесплатное заполнение
+achievement.thermalexpansion.transposer.desc=Наполнять вёдра руками - тяжкий труд.
+achievement.thermalexpansion.precipitator=Битва снежками!
 achievement.thermalexpansion.precipitator.desc=Не делает жёлтый снег.
-achievement.thermalexpansion.extruder=Эта вещица делает камни!
-achievement.thermalexpansion.extruder.desc=А также обсидиан!
+achievement.thermalexpansion.extruder=Эта машина камниТ!
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 3]
```diff
@@ -1,63 +1,63 @@
...
+achievement.thermalexpansion.extruder.desc=А также обсидианиТ!
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 4]
```diff
@@ -1,63 +1,63 @@
...
 achievement.thermalexpansion.accumulator=Вода, вода везде
-achievement.thermalexpansion.accumulator.desc=Нет, он не работает с лавой. Вы монстр.
+achievement.thermalexpansion.accumulator.desc=Нет, она не работает с лавой. Вы монстр.
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 5]
```diff
@@ -1,63 +1,63 @@
...
 achievement.thermalexpansion.assembler=Порочный цикл
-achievement.thermalexpansion.assembler.desc=Нет, он не создаёт это.
+achievement.thermalexpansion.assembler.desc=Нет, она его не делает.
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 6]
```diff
@@ -1,63 +1,63 @@
...
 achievement.thermalexpansion.charger=Что за Флакс?
-achievement.thermalexpansion.charger.desc=Флакс не знает границ. Поэтому берегите свои руки.
+achievement.thermalexpansion.charger.desc=Флакс не знает границ. Так что берегите руки.
 achievement.thermalexpansion.insolator=Солнце в коробке!
-achievement.thermalexpansion.insolator.desc=Точно неорганическое.
+achievement.thermalexpansion.insolator.desc=Точно не органическое.
 
 achievement.thermalexpansion.wrench=Выглядит как гвоздь!
-achievement.thermalexpansion.wrench.desc=Любой инструмент - молоток, если Вам он *действительно* нужен.
+achievement.thermalexpansion.wrench.desc=Любой инструмент - молоток, если вам это *действительно* нужно.
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 7]
```diff
@@ -1,63 +1,63 @@
...
+achievement.thermalexpansion.florb=Я могу бросить это?!
+achievement.thermalexpansion.florb.desc=Содержит жидкость. Вы можете его бросить.
+achievement.thermalexpansion.florbMagmatic=Смотрите как Мир горит...
+achievement.thermalexpansion.florbMagmatic.desc=Шар. Да, в нём лава. Вы монстр.
 
 achievement.thermalexpansion.capacitorPotato=Полусырая идея!
 achievement.thermalexpansion.capacitorPotato.desc=Кар-то-шка
 
-achievement.thermalexpansion.fluidRedstone=Красная революция!
+achievement.thermalexpansion.fluidRedstone=Рэдстоун революция!
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 8]
```diff
@@ -1,63 +1,63 @@
...
 achievement.thermalexpansion.fluidGlowstone.desc=Вот как происходят эти образования! Всё это ерунда!
-achievement.thermalexpansion.fluidEnder=Wibbly-Wobbly, Timey-Wimey.
-achievement.thermalexpansion.fluidEnder.desc=Конец на самом деле только начало.
+achievement.thermalexpansion.fluidEnder=Вибли-вобли, Тими-вими.
+achievement.thermalexpansion.fluidEnder.desc=Край на самом деле, только начало.
 achievement.thermalexpansion.fluidPyrotheum=О, БОЖЕ ПОЧЕМУ?!
-achievement.thermalexpansion.fluidPyrotheum.desc=На самом деле он хуже, чем лава. Вы монстр.
-achievement.thermalexpansion.fluidCryotheum=Зима приближается
-achievement.thermalexpansion.fluidCryotheum.desc=Он также хуже, чем лава, за исключением того, что он - холод. Вы йети.
+achievement.thermalexpansion.fluidPyrotheum.desc=Это на самом деле хуже, чем лава. Вы монстр.
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 9]
```diff
@@ -1,63 +1,63 @@
...
+achievement.thermalexpansion.fluidCryotheum=Зима близко
+achievement.thermalexpansion.fluidCryotheum.desc=Это также хуже, чем лава, только оно холодное. Вы йети.
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 10]
```diff
@@ -75,11 +75,11 @@ info.thermalexpansion.maxIn=Макс. вход
 info.thermalexpansion.maxOut=Макс. выход
 
 #Workbench
-info.thermalexpansion.gridClear=Очистить сетку создания
-info.thermalexpansion.gridSet=Установить сетку создания из схемы
+info.thermalexpansion.gridClear=Очистить сетку рецепта
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 11]
```diff
@@ -119,15 +119,15 @@ info.thermalexpansion.toggleInsolatorUnlock=Открыть слот удобре
...
+info.thermalexpansion.intensityImpulse=Сила запуска
 info.thermalexpansion.duration=Длительность
 
 #Tier Localization
-info.thermalexpansion.creative=Творческ.
-info.thermalexpansion.basic=Основн.
-info.thermalexpansion.hardened=Усилен.
-info.thermalexpansion.reinforced=Укреплён.
-info.thermalexpansion.resonant=Резонирующ.
+info.thermalexpansion.creative=Творческий
```
Comment (Adaptivity) [inline ru_RU.lang]: No for all tiers.

File: ru_RU.lang [hunk 12]
```diff
@@ -136,6 +136,7 @@ info.thermalexpansion.augment.fuelConsumed=Топлива потреблено
 info.thermalexpansion.augment.fuelEnergy=Основная энергия топлива
 info.thermalexpansion.augment.levels.0=Требуется, чтобы предыдущие уровни были установлены.
 info.thermalexpansion.augment.levels.1=Используется высочайший уровень расширения.
+info.thermalexpansion.augment.levels.2=Требуется установленный рэдстоун контроль
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 13]
```diff
@@ -145,29 +146,29 @@ info.thermalexpansion.augment.speed=Скорость
...
 
-info.thermalexpansion.augment.dynamoCoilDuct=Генератор: Сторонняя доступность
+info.thermalexpansion.augment.dynamoCoilDuct=Генератор: Доступ по сторонам
 info.thermalexpansion.augment.dynamoCoilDuct.0=Позволяет вводить/выводить предметы и жидкости через катушку генератора.
 info.thermalexpansion.augment.dynamoEfficiency=Генератор: Эффективность топлива
-info.thermalexpansion.augment.dynamoOutput=Генератор: Выходная энергия
-info.thermalexpansion.augment.dynamoOutput.0=Дополнительный RF/т
-info.thermalexpansion.augment.dynamoThrottle=Генератор: Продвинутое регулирование
+info.thermalexpansion.augment.dynamoOutput=Генератор: Выход энергии
+info.thermalexpansion.augment.dynamoOutput.0=Дополнительные РФ/т
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 14]
```diff
@@ -145,29 +146,29 @@ info.thermalexpansion.augment.speed=Скорость
...
-info.thermalexpansion.augment.generalAutoTransfer=Автоматический выход
+info.thermalexpansion.augment.generalAutoTransfer=Автоматический вывод
 info.thermalexpansion.augment.generalAutoTransfer.0=Позволяет предметам и жидкостям автоматически выводиться из устройства.
 info.thermalexpansion.augment.generalReconfigSides=Настраиваемые стороны
-info.thermalexpansion.augment.generalReconfigSides.0=Позволяет заново настроить стороны устройства.
-info.thermalexpansion.augment.generalRedstoneControl=Красный контроль
-info.thermalexpansion.augment.generalRedstoneControl.0=Позволяет установить на устройство настройки контроля красным камнем.
+info.thermalexpansion.augment.generalReconfigSides.0=Позволяет настроить поведение сторон устройства.
+info.thermalexpansion.augment.generalRedstoneControl=Рэдстоун контроль
+info.thermalexpansion.augment.generalRedstoneControl.0=Позволяет установить на машине контроль рэдстоуном.
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 15]
```diff
@@ -145,29 +146,29 @@ info.thermalexpansion.augment.speed=Скорость
...
 
-info.thermalexpansion.augment.generalAutoTransfer=Автоматический выход
+info.thermalexpansion.augment.generalAutoTransfer=Автоматический вывод
 info.thermalexpansion.augment.generalAutoTransfer.0=Позволяет предметам и жидкостям автоматически выводиться из устройства.
 info.thermalexpansion.augment.generalReconfigSides=Настраиваемые стороны
-info.thermalexpansion.augment.generalReconfigSides.0=Позволяет заново настроить стороны устройства.
-info.thermalexpansion.augment.generalRedstoneControl=Красный контроль
-info.thermalexpansion.augment.generalRedstoneControl.0=Позволяет установить на устройство настройки контроля красным камнем.
+info.thermalexpansion.augment.generalReconfigSides.0=Позволяет настроить поведение сторон устройства.
+info.thermalexpansion.augment.generalRedstoneControl=Рэдстоун контроль
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 16]
```diff
@@ -145,29 +146,29 @@ info.thermalexpansion.augment.speed=Скорость
...
+info.thermalexpansion.augment.generalRedstoneControl.0=Позволяет установить на машине контроль рэдстоуном.
 
 info.thermalexpansion.augment.machineSecondary=Механизм: Вторичный выход
 info.thermalexpansion.augment.machineSpeed=Механизм: Скорость обработки
 
-info.thermalexpansion.augment.machineFurnaceFood=Красная печь: Специализация
+info.thermalexpansion.augment.machineFurnaceFood=Рэдстоун печь: Специализация
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 17]
```diff
@@ -176,41 +177,41 @@ info.thermalexpansion.capacitor.3=Используйте сидя, чтобы д
...
+info.thermalexpansion.device.activator=Функций больше, чем может объяснить эта подсказка.
+info.thermalexpansion.device.breaker=Ломает блоки. Довольно просто.
+info.thermalexpansion.device.nullifier=Делает вещи несуществующими.
 
 info.thermalexpansion.diagram.0=Используйте сидя, чтобы стереть.
 info.thermalexpansion.diagram.1=Используйте на устройстве, чтобы скопировать его настройки.
 info.thermalexpansion.diagram.2=Используйте на устройстве, чтобы передать записанные настройки.
 
-info.thermalexpansion.dynamo.generate=Генерирует энергию Redstone Flux.
+info.thermalexpansion.dynamo.generate=Генерирует энергию (Рэдстоун Флакс).
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 18]
```diff
@@ -176,41 +177,41 @@ info.thermalexpansion.capacitor.3=Используйте сидя, чтобы д
...
 info.thermalexpansion.machine.crucible=Расплавляет предметы в жидкости.
 info.thermalexpansion.machine.transposer=Управляет жидкостями и жидкостными контейнерами.
 info.thermalexpansion.machine.precipitator=Замораживает жидкости в твёрдые материалы.
 info.thermalexpansion.machine.extruder=Смешивает горячие и холодные жидкости.
-info.thermalexpansion.machine.accumulator=Тянет ближайшую воду.
+info.thermalexpansion.machine.accumulator=Всасывает воду поблизости.
 info.thermalexpansion.machine.assembler=Создаёт предметы.
-info.thermalexpansion.machine.charger=Заряжает предметы энергией RF.
-info.thermalexpansion.machine.insolator=Наполняет растения энергией 'солнца' и RF.
+info.thermalexpansion.machine.charger=Заряжает предметы энергией РФ.
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 19]
```diff
@@ -176,41 +177,41 @@ info.thermalexpansion.capacitor.3=Используйте сидя, чтобы д
...
+info.thermalexpansion.machine.insolator=Наполняет растения энергией 'солнца' и РФ.
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 20]
```diff
@@ -218,37 +219,43 @@ info.thermalexpansion.plate.translocate=Телепортирует вещи пр
...
-info.thermalexpansion.florb.5=Шарик будет лететь дальше.
+info.thermalexpansion.florb.5=Шар пролетит дальше.
 
 info.thermalexpansion.tool.battleWrench=На 90% молот, на 10% ключ.
+info.thermalexpansion.tool.chiller=Охлаждает на расстоянии.
+info.thermalexpansion.tool.igniter=Поджигает на расстоянии.
+info.thermalexpansion.tool.multimeter.0=Предоставляет информацию о машинах/устройствах.
+info.thermalexpansion.tool.multimeter.1=Используйте сидя, чтобы настроить некоторые их них.
+info.thermalexpansion.tool.pump=Закачивает жидкости в/из поддерживеаемых контейнеров.
+info.thermalexpansion.tool.transfuser=Передаёт РФ в/из поддерживаемых вещей и устройств.
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 21]
```diff
@@ -218,37 +219,43 @@ info.thermalexpansion.plate.translocate=Телепортирует вещи пр
...
 item.thermalexpansion.augment.dynamoOutput0.name=Вторичная отдающая катушка
 item.thermalexpansion.augment.dynamoOutput1.name=Усилитель модуляции флакса
 item.thermalexpansion.augment.dynamoOutput2.name=Флаксовый усилитель криогенной катушки
-item.thermalexpansion.augment.dynamoThrottle.name=Ограничитель поля намагничивания
 
 item.thermalexpansion.augment.generalAutoTransfer.name=Интегрированный сервомеханизм
-item.thermalexpansion.augment.generalReconfigSides.name=Интегрированная модульная структура
-item.thermalexpansion.augment.generalRedstoneControl.name=Интегрированная красная схема
+item.thermalexpansion.augment.generalReconfigSides.name=Интегрированный модульный каркас
+item.thermalexpansion.augment.generalRedstoneControl.name=Интегрированная редстоун микросхема
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 22]
```diff
@@ -256,28 +263,28 @@ item.thermalexpansion.augment.machineSpeed2.name=Пространственно-
...
+item.thermalexpansion.augment.machineExtruderBoost0.name=Ускоренное вытеснение
+item.thermalexpansion.augment.machineExtruderBoost1.name=Магматический катализатор
 item.thermalexpansion.augment.machineExtruderBoost2.name=Пирокластическое генерирование
 
 item.thermalexpansion.capacitor.creative.name=Творческий флаксовый конденсатор
 item.thermalexpansion.capacitor.potato.name=Неровный флаксовый конденсатор
 item.thermalexpansion.capacitor.basic.name=Свинцовый флаксовый конденсатор
 item.thermalexpansion.capacitor.hardened.name=Усиленный флаксовый конденсатор
-item.thermalexpansion.capacitor.reinforced.name=Красный флаксовый конденсатор
+item.thermalexpansion.capacitor.reinforced.name=Рэдстоун флаксовый конденсатор
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 23]
```diff
@@ -256,28 +263,28 @@ item.thermalexpansion.augment.machineSpeed2.name=Пространственно-
...
-item.thermalexpansion.florb.florbMagmatic.name=Магматический шарик
+item.thermalexpansion.florb.florb.name=Шар
+item.thermalexpansion.florb.florbMagmatic.name=Магматический шар
 
 item.thermalexpansion.material.pneumaticServo.name=Пневматический сервомотор
-item.thermalexpansion.material.powerCoilGold.name=Красная принимающая катушка
-item.thermalexpansion.material.powerCoilSilver.name=Красная отдающая катушка
-item.thermalexpansion.material.powerCoilElectrum.name=Красная проводящая катушка
-item.thermalexpansion.material.lock.name=Сигналовый замок безопасности
+item.thermalexpansion.material.powerCoilGold.name=Редстоун принимающая катушка
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 24]
```diff
@@ -256,28 +263,28 @@ item.thermalexpansion.augment.machineSpeed2.name=Пространственно-
...
+item.thermalexpansion.material.powerCoilSilver.name=Рэдстоун отдающая катушка
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 25]
```diff
@@ -256,28 +263,28 @@ item.thermalexpansion.augment.machineSpeed2.name=Пространственно-
...
+item.thermalexpansion.material.powerCoilElectrum.name=Рэдстоун проводящая катушка
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 26]
```diff
@@ -256,28 +263,28 @@ item.thermalexpansion.augment.machineSpeed2.name=Пространственно-
...
+item.thermalexpansion.material.lock.name=Замок безопасности
```
Comment (Adaptivity) [inline ru_RU.lang]: *Синаловый*

File: ru_RU.lang [hunk 27]
```diff
@@ -310,37 +320,37 @@ recipe.thermalexpansion.upgrade=Создание улучшений
 recipe.thermalexpansion.secure=Создание безопасности
 
 #Info Tabs
-tab.thermalexpansion.cell.0=Содержит Redstone Flux.
-tab.thermalexpansion.cell.1=Удерживайте Shift или Ctrl, для тонкой настройки контроля энергии.
+tab.thermalexpansion.cell.0=Содержит Рэдстоун Флакс.
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 28]
```diff
@@ -310,37 +320,37 @@ recipe.thermalexpansion.upgrade=Создание улучшений
...
 tab.thermalexpansion.device.activator.2=Обманчивая универсальность!
-tab.thermalexpansion.device.breaker.0=Ломает все вещи!
+tab.thermalexpansion.device.breaker.0=Ломает всё!
 tab.thermalexpansion.device.breaker.1=Держите лицо подальше от передней части устройства.
 tab.thermalexpansion.device.nullifier.0=Отправляет все вещи в /dev/null!
 tab.thermalexpansion.device.nullifier.1=Это ужасное место, даже жидкости не могут убежать.
 
 tab.thermalexpansion.dynamo.0=Скорость генерации меняется в зависимости от потребности в энергии.
-tab.thermalexpansion.dynamo.steam.0=Генерирует Redstone Flux, используя пар.
+tab.thermalexpansion.dynamo.steam.0=Генерирует Рэдстоун Флакс, используя пар.
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 29]
```diff
@@ -310,37 +320,37 @@ recipe.thermalexpansion.upgrade=Создание улучшений
...
 tab.thermalexpansion.dynamo.steam.1=Твёрдое топливо и вода могут быть использованы для создания пара.
-tab.thermalexpansion.dynamo.magmatic.0=Генерирует Redstone Flux, используя очень горячие жидкости.
-tab.thermalexpansion.dynamo.compression.0=Генерирует Redstone Flux, используя жидкое топливо и хладагенты.
-tab.thermalexpansion.dynamo.reactant.0=Генерирует Redstone Flux, используя алхимические реакции.
+tab.thermalexpansion.dynamo.magmatic.0=Генерирует Рэдстоун Флакс, используя очень горячие жидкости.
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 30]
```diff
@@ -310,37 +320,37 @@ recipe.thermalexpansion.upgrade=Создание улучшений
...
+tab.thermalexpansion.dynamo.compression.0=Генерирует Рэдстоун Флакс, используя жидкое топливо и хладагенты.
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 31]
```diff
@@ -310,37 +320,37 @@ recipe.thermalexpansion.upgrade=Создание улучшений
...
+tab.thermalexpansion.dynamo.reactant.0=Генерирует Рэдстоун Флакс, используя алхимические реакции.
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 32]
```diff
@@ -310,37 +320,37 @@ recipe.thermalexpansion.upgrade=Создание улучшений
...
 tab.thermalexpansion.dynamo.reactant.1=Требуется твёрдый реагент и жидкое топливо.
-tab.thermalexpansion.dynamo.enervation.0=Извлекает Redstone Flux напрямую из природных источников или из объектов, которые содержат его.
+tab.thermalexpansion.dynamo.enervation.0=Извлекает Рэдстоун Флакс напрямую из природных источников или из объектов, которые содержат его.
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 33]
```diff
@@ -310,37 +320,37 @@ recipe.thermalexpansion.upgrade=Создание улучшений
...
 
-tab.thermalexpansion.ender.tesseract.0=Используйте его, чтобы мгновенно транспортировать предметы на огромные расстояния или через измерения.
-tab.thermalexpansion.ender.tesseract.1=Настройте частоту Края, чтобы определить соединения.
+tab.thermalexpansion.ender.tesseract.0=Используйте его, чтобы мгновенно перемещать предметы на огромные расстояния или через измерения.
+tab.thermalexpansion.ender.tesseract.1=Настройте частоту Края, чтобы создать соединения.
 
-tab.thermalexpansion.light.0=Настраиваемый источник света, чувствителен к красному камню.
+tab.thermalexpansion.light.0=Настраиваемый источник света, чувствителен к рэдстоуну.
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 34]
```diff
@@ -310,37 +320,37 @@ recipe.thermalexpansion.upgrade=Создание улучшений
...
 tab.thermalexpansion.light.1=Цвет и режим можно регулировать ползунами.
-tab.thermalexpansion.light.2=Заменяет тьму светом!
+tab.thermalexpansion.light.2=Замените тьму весельем!
 
-tab.thermalexpansion.machine.furnace.0=Плавит вещи, используя Redstone Flux.
+tab.thermalexpansion.machine.furnace.0=Плавит вещи, используя Рэдстоун Флакс.
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 35]
```diff
@@ -352,45 +362,45 @@ tab.thermalexpansion.machine.sawmill.2=Следите за своими рука
...
+tab.thermalexpansion.machine.smelter.3=Не будет готовить еду.
 tab.thermalexpansion.machine.crucible.0=Превращает твёрдые материалы в жидкости!
 tab.thermalexpansion.machine.crucible.1=Очень полезен для строительства продвинутых устройств.
 tab.thermalexpansion.machine.crucible.2=Держите вдали от детей и домашних животных.
 tab.thermalexpansion.machine.transposer.0=Заполняет или опустошает вещи, которые содержат жидкости.
 tab.thermalexpansion.machine.transposer.1=Нажмите кнопку для переключения между режимами "Заполнять" и "Извлекать".
-tab.thermalexpansion.machine.transposer.2=Обычно обратимы.
-tab.thermalexpansion.machine.precipitator.0=Использует Redstone Flux, чтобы замораживать разные жидкости!
+tab.thermalexpansion.machine.transposer.2=Обычно операции обратимы.
+tab.thermalexpansion.machine.precipitator.0=Использует Рэдстоун Флакс, чтобы замораживать разные жидкости!
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 36]
```diff
@@ -352,45 +362,45 @@ tab.thermalexpansion.machine.sawmill.2=Следите за своими рука
...
-tab.thermalexpansion.machine.accumulator.0=Забирает ближайшую воду.
+tab.thermalexpansion.machine.accumulator.0=Выкачивает ближайшую воду.
 tab.thermalexpansion.machine.accumulator.1=Поместите в лужу воды, чтобы ускорить этот процесс.
-tab.thermalexpansion.machine.accumulator.2=Не тонет.
-tab.thermalexpansion.machine.assembler.0=Создаёт рецепты сохранённые в схемах.
-tab.thermalexpansion.machine.assembler.1=Использует 20 Redstone Flux за предмет.
-tab.thermalexpansion.machine.assembler.2=Нет, он не будет делать это.
+tab.thermalexpansion.machine.accumulator.2=Не утоните.
+tab.thermalexpansion.machine.assembler.0=Создаёт рецепты сохранённые в чертежах.
+tab.thermalexpansion.machine.assembler.1=Использует 20 Рэдстоун Флакс за предмет.
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 37]
```diff
@@ -352,45 +362,45 @@ tab.thermalexpansion.machine.sawmill.2=Следите за своими рука
...
+tab.thermalexpansion.machine.assembler.2=Нет, он не будет делать то, о чём вы только что подумали.
 tab.thermalexpansion.machine.charger.0=Наполняет энергией совместимые предметы.
-tab.thermalexpansion.machine.charger.1=Размер заряда может быть ограничен для предмета.
+tab.thermalexpansion.machine.charger.1=Скорость заряда может быть ограничена для предмета.
 tab.thermalexpansion.machine.charger.2=Почувствуйте гул.
-tab.thermalexpansion.machine.insolator.0=Наполняет растения питательными веществами и светом, используя Redstone Flux.
+tab.thermalexpansion.machine.insolator.0=Наполняет растения питательными веществами и светом, используя Рэдстоун Флакс.
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 38]
```diff
@@ -352,45 +362,45 @@ tab.thermalexpansion.machine.sawmill.2=Следите за своими рука
...
 tab.thermalexpansion.machine.insolator.1=Не совсем 'органическое' фермерство.
 tab.thermalexpansion.machine.insolator.2=Левый слот может быть закрыт, чтобы хранить только определённые предметы.
 tab.thermalexpansion.machine.insolator.3=Слава лампе накаливания!
 
-tab.thermalexpansion.plate.signal.0=Излучает удалённый сигнал красного камня при контакте.
+tab.thermalexpansion.plate.signal.0=Излучает удалённый сигнал рэдстоуна при контакте.
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 39]
```diff
@@ -406,12 +416,12 @@ tile.thermalexpansion.cell=Энергетическая ячейка
 tile.thermalexpansion.cell.creative.name=Творческая энергетическая ячейка
 tile.thermalexpansion.cell.basic.name=Свинцовая энергетическая ячейка
 tile.thermalexpansion.cell.hardened.name=Усиленная энергетическая ячейка
-tile.thermalexpansion.cell.reinforced.name=Красная энергетическая ячейка
+tile.thermalexpansion.cell.reinforced.name=Рэдстоун энергетическая ячейка
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 40]
```diff
@@ -423,35 +433,36 @@ tile.thermalexpansion.dynamo.enervation.name=Ослабленный генера
...
-tile.thermalexpansion.frame.tesseractEmpty.name=Рамка тессеракта (Пустая)
-tile.thermalexpansion.frame.tesseractFull.name=Рамка тессеракта (Полная)
-tile.thermalexpansion.frame.illuminator.name=Рамка светильника
+tile.thermalexpansion.frame.machineBasic.name=Рама механизма (Основная)
+tile.thermalexpansion.frame.machineHardened.name=Рама механизма (Усиленная)
+tile.thermalexpansion.frame.machineReinforced.name=Рама механизма (Укреплённая)
+tile.thermalexpansion.frame.machineResonant.name=Рама механизма (Резонирующая)
+tile.thermalexpansion.frame.cellBasic.name=Рама свинцовой энергетической ячейки
+tile.thermalexpansion.frame.cellHardened.name=Рама усиленной энергетической ячейки
+tile.thermalexpansion.frame.cellReinforcedEmpty.name=Рама рэдстоун энергетической ячейки (Пустая)
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 41]
```diff
@@ -423,35 +433,36 @@ tile.thermalexpansion.dynamo.enervation.name=Ослабленный генера
...
+tile.thermalexpansion.frame.cellReinforcedFull.name=Рама рэдстоун энергетической ячейки (Полная)
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 42]
```diff
@@ -423,35 +433,36 @@ tile.thermalexpansion.dynamo.enervation.name=Ослабленный генера
...
 tile.thermalexpansion.glass.name=Закалённое стекло
++tile.thermalexpansion.glassLumium.name=Закалённое ламиумное стекло
 
 tile.thermalexpansion.light.name=6-режимное освещение
 tile.thermalexpansion.light.illuminator.name=Светильник из светящегося камня
 tile.thermalexpansion.light.lampLumiumRadiant.name=Сияющий ламиумовый светильник
 tile.thermalexpansion.light.lampLumium.name=Ламиумовый светильник
 
-tile.thermalexpansion.machine.furnace.name=Красная печь
+tile.thermalexpansion.machine.furnace.name=Рэдстоун печь
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 43]
```diff
@@ -459,7 +470,7 @@ tile.thermalexpansion.machine.insolator.name=Фитогенный светиль
 
 tile.thermalexpansion.plate.frame.name=Рамка пластины
 tile.thermalexpansion.plate.impulse.name=Импульсная пластина
-tile.thermalexpansion.plate.signal.name=Красная пластина
+tile.thermalexpansion.plate.signal.name=Рэдстоун пластина
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

File: ru_RU.lang [hunk 44]
```diff
@@ -423,35 +433,36 @@ tile.thermalexpansion.dynamo.enervation.name=Ослабленный генера
...
 tile.thermalexpansion.light.illuminator.name=Светильник из светящегося камня
 tile.thermalexpansion.light.lampLumiumRadiant.name=Сияющий ламиумовый светильник
 tile.thermalexpansion.light.lampLumium.name=Ламиумовый светильник
 
-tile.thermalexpansion.machine.furnace.name=Красная печь
+tile.thermalexpansion.machine.furnace.name=Рэдстоун печь
 tile.thermalexpansion.machine.pulverizer.name=Измельчитель
 tile.thermalexpansion.machine.sawmill.name=Лесопилка
-tile.thermalexpansion.machine.smelter.name=Индукционная плавильня
+tile.thermalexpansion.machine.smelter.name=Индукционная печь
```
Comment (Adaptivity) [inline ru_RU.lang]: No.

**Comment by Adaptivity** [comment]
*Potentially* I can apply *some* changes in the next update. But not everything.

**Comment by KingLemming** [comment]
@Adaptivity is our trusted source on this one. Feel free to confer with him, but he does this sort of thing for a TON of mods and I'm going with his call on it.

**Pull request closed without merging**